example/app/views/layouts/embedded_app.html.erb

```diff
@@ -20,14 +20,11 @@
 
     <script src="https://cdn.shopify.com/s/assets/external/app.js?<%= Time.now.strftime('%Y%m%d%H') %>"></script>
 
-    <script type="text/javascript">
-      ShopifyApp.init({
-        apiKey: "<%= ShopifyApp.configuration.api_key %>",
-        shopOrigin: "<%= "https://#{ @shop_session.url }" if @shop_session %>",
-        debug: false,
-        forceRedirect: true
-      });
-    </script>
+    <%= content_tag(:div, nil, id: 'shopify-app-init', data: {
+      api_key: ShopifyApp.configuration.api_key,
+      shop_origin: ("https://#{ @shop_session.url }" if @shop_session),
+      debug: (Rails.env.development? ? 'true' : 'false')
```

Comment:
I don't think this should be a string otherwise it will always evaluate to true:
```suggestion
      debug: !!Rails.env.development?
```

Reply:
I expect it's already boolean, so is the `!!` even needed?